src/wanna/core/services/notebook.py

```diff
@@ -642,12 +642,12 @@ def _get_jupyterlab_link(self, instance_id: str) -> str:
         instance_info = self.notebook_client.get_runtime({"name": instance_id})
         return f"https://{instance_info.access_config.proxy_uri}"
 
-    def _return_diff(self):
+    def _return_diff(self) -> Tuple[List[Any], List[Any]]:
```

Comment:
Is not returning Any, is returning some object, please add the correct type. thx